BrownField/Networking/Concepts/Connectivity-Multi-region-with-vWAN/Dual-Region-with-vWAN.md

```diff
@@ -0,0 +1,122 @@
+
+**Traffic Flow**
+| From |   To |  Hub 1 VNets | Hub 2 VNets | AVS Region 1| AVS Region 2| 
+| -------------- | -------- | ---------- | ---| ---| ---|
+| On-Premise    | &#8594;| Hub1Fw>Vnet1|  Hub2Fw>Vnet2  | Global Reach(A)>AVS Cloud Region 1 | Global Reach(B)>AVS Cloud Region 2| 
+
+### Azure Virtual Network connectivity & traffic flow
+
+This section will focus only on connectivity from an Azure Vnet perspective. As shown in the diagram below, both Vnet1 and Vnet2 will have a vnet peering directly to its local regional hub. 
+
+The diagram shows how all Azure native resources in Vnet1 and Vnet2 will learn routes under their "Effective Routes". A Secure Hub with Routing Intent enabled will always send the default RFC 1918 addresses (10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16) to peered Vnets. In our case, since Routing Intent is enabled, all resources within Vnet1 and Vnet2 will have the default RFC 1918 address with a next-hop of their local regional hub firewall. All traffic ingressing and egressing the Vnets will always transit the Hub Firewalls. Please see the traffic flow below for more detailed information.
+
+![image](https://github.com/jasonamedina/Enterprise-Scale-for-AVS/assets/97964083/5f153c7b-f683-44e7-b9ab-cebece766580)
+
+**Traffic Flow**
+| From |   To |  On-Premise | AVS Region 1 | AVS Region 2| Cross-Region Vnet| 
+| -------------- | -------- | ---------- | ---| ---| ---|
+| Vnet1    | &#8594;| Hub1Fw>OnPremise|  Hub1Fw>AVS Cloud Region 1  | Hub1Fw>AVS Cloud Region 2 | Hub1Fw>Hub2Fw>Vnet2 |
+| Vnet2    | &#8594;| Hub2Fw>OnPremise|  Hub2Fw>AVS Cloud Region 1  | Hub2Fw>AVS Cloud Region 2 | Hub2Fw>Hub1Fw>Vnet1 |
+
+### Internet connectivity
+
+This section will focus only on how internet connectivity will be provided for Azure native resources in Vnets and AVS Private Clouds in both regions.
+
+As mentioned earlier, when you enable Routing Intent on the Secure Hub, it will advertise RFC 1918 to all peered Vnets. However, you also have the option to advertise a default route 0.0.0.0/0 to provide internet connectivity for downstream resources. The blue default route originated in Hub1 with a next hop of HubFw1 and the red default route originated in Hub2 with a next hop of HubFw2.
+
+ Each Vnet will egress to the internet using its local regional hub firewall. The default route is never advertised across regional hubs over the "inter-hub" link. Therefore, Vnets can only use their local regional hub for internet access. 
+
+From an AVS Private Cloud perspective, when advertising the default route across regional connections (pink connections), you need to configure route maps with BGP prepending on the Secure vWAN hubs. If you do not use BGP prepending AVS Cloud regions will load balance internet traffic between its local and regional hub. This load balance would introduce asymmetric traffic and impact internet performance. 
+
```

Comment:
Did you test AS Path Prep with public or private ASN's? I am asking because of this documented caveat: "The key point is that you must prepend Public ASN numbers to influence how Azure VMware Solution routes traffic back to on-premises. If you prepend using Private ASN, Azure VMware Solution will ignore the prepend, and the ECMP behavior above will occur. Even if you operate a Private BGP ASN on-premises, it's still possible to configure your on-premises devices to utilize a Public ASN when prepending routes outbound, to ensure compatibility with Azure VMware Solution." (https://learn.microsoft.com/en-us/azure/azure-vmware/concepts-network-design-considerations#prerequisites). I ***suspect*** that it only applies to routes propagated to AVS over Global Reach connections, but I would recommend checking with Kurt if using private ASNs for Path Prep over connections between ER gateways and managed circuits is OK

Reply:
@fguerri, we covered this together today, and I went over a review regarding my findings on how Private ASN BGP prepending works fine and will not impact the design. I also added a note under the "Internet Connectivity" section to reflect this. Please see the highlighted sentence below. 

![image](https://github.com/Azure/Enterprise-Scale-for-AVS/assets/97964083/c00092f4-9e7c-48ee-b054-0872a2846d2a)